docs/src/whatsnew/latest.rst

```diff
@@ -94,6 +95,8 @@ This document explains the changes made to Iris for this release
    for the outstanding work of `@ed-hawkins`_ et al for `#ShowYourStripes`_.
    (:pull:`5141`)
 
+#. `@HGWright`_ fixed some typo's from Gitwash. (:pull:`5145`)
+
```

Comment:
Minor point, but we have **two** blank lines between sections to help the developers eye when writing the docs